### ISSUE TITLE
Loss Function

### ISSUE THREAD

**Issue by ajay960singh** [vicwer/C3AE_Age_Estimation#14]
It seems that the loss function is a weighted sum of KL Divergence and L1 loss. I'm just wondering why cross-entropy was not used as a loss function instead. Can you pls explain?
